src/MuTalk-Tests/MutationTestingAnalysisBuilderTest.class.st

```diff
@@ -0,0 +1,59 @@
+Class {
+	#name : 'MutationTestingAnalysisBuilderTest',
+	#superclass : 'TestCase',
+	#category : 'MuTalk-Tests',
+	#package : 'MuTalk-Tests'
+}
+
+{ #category : 'tests' }
+MutationTestingAnalysisBuilderTest >> testCustomParameters [
+
+	| analysis |
+	analysis := MutationTestingAnalysisBuilder new
+		            testClasses:
+			            { AuxiliarClassForMutationTestingAnalysisTest };
+		            classesToMutate:
+			            { AuxiliarClassForMutationTestingAnalysis };
+		            operators:
+			            (Set with: ReplaceIfTrueReceiverWithTrueOperator new);
+		            mutantSelectionStrategy:
+			            RandomMutantSelectionStrategy new;
+		            testSelectionStrategy:
+			            SelectingFromCoverageMutantSelectionStrategy new;
+		            budget: (MTTimeBudget for: 10 seconds);
+		            buildAnalysis.
+	self
+		assert: analysis modelClasses
+		equals: (Array with: AuxiliarClassForMutationTestingAnalysis).
+	self
+		assert: (Set withAll: (analysis testCases collect: #method))
+		equals:
+		(Set withAll: AuxiliarClassForMutationTestingAnalysisTest methods).
+	self
+		assert: (analysis operators collect: [ :op | op species ])
+		equals: (Set with: ReplaceIfTrueReceiverWithTrueOperator).
+	self
+		assert: analysis mutantSelectionStrategy species
+		equals: RandomMutantSelectionStrategy.
+	self
+		assert: analysis testSelectionStrategy species
+		equals: SelectingFromCoverageMutantSelectionStrategy.
+	self assert: analysis budget species equals: MTTimeBudget
```

Comment:
Hmm isn't this implicitly tested by the other tests (e.g., the ones that test the strategies)?
I mean, how is this not just testing the setter/getter pairs?